packages/php-wasm/compile/bison2.7/Dockerfile

```diff
@@ -1,11 +1,15 @@
 FROM playground-php-wasm:base
 
 COPY ./bison2.7/bison27.patch /root/bison27.patch
-RUN wget http://ftp.gnu.org/gnu/bison/bison-2.7.tar.gz && \
-    tar -xvf bison-2.7.tar.gz && \
-    rm bison-2.7.tar.gz && \
-    cd bison-2.7 && \
-    git apply --no-index /root/bison27.patch && \
-    ./configure --prefix=/usr/local/bison --with-libiconv-prefix=/usr/local/libiconv/ && \
-    make && \
-    make install
+RUN set -euxo pipefail; \
+	wget http://ftp.gnu.org/gnu/bison/bison-2.7.tar.gz; \
```

Comment:
This is a `HTTP` URL, not an `HTTPS` one. I think that's what the trunk does today, which is unfortunate, but since this PR touches this part of the code it could update it (like we discussed in the other thread).

Reply:
Any thoughts on this one?

Reply:
Ah, I'd gotten that from https://github.com/WordPress/wordpress-playground/blob/trunk/packages/php-wasm/compile/bison2.7/Dockerfile#L3-L5